### PR TITLE
rankscore for genmod as reference and rule

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Added:
- rankscore init file as reference
- rankscore rule to rank tumor-normal and tumor-only panel variants

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by hassanfa** [comment]
TN panel:
image

T_panel:
image

File: BALSAMIC/workflows/balsamic.smk [hunk 1]
```diff
@@ -187,6 +187,8 @@ if config["analysis"]["sequencing_type"] == "wgs":
                                      "snakemake_rules/variant_calling/cnvkit_single.rule"])
 else:
     sentieon_callers = ["tnhaplotyper"] if sentieon else []
+    annotation_rules.append("snakemake_rules/annotation/rankscore.rule")
```
Comment (ashwini06) [inline BALSAMIC/workflows/balsamic.smk:190]: Can't find this `rankscore.rule` in this PR?

File: BALSAMIC/workflows/balsamic.smk [hunk 2]
```diff
@@ -260,7 +262,7 @@ if config['analysis']["analysis_type"] in ["paired", "single"]:
 if config['analysis']["analysis_type"] in ["paired", "single"] and config["analysis"]["sequencing_type"] != "wgs":
     analysis_specific_results.extend(expand(vep_dir + "{vcf}.pass.balsamic_stat",
                                             vcf=get_vcf(config, ["vardict"], [config["analysis"]["case_id"]])))
-    analysis_specific_results.extend([expand(vep_dir + "{vcf}.all.filtered.vcf.gz",
+    analysis_specific_results.extend([expand(vep_dir + "{vcf}.all.filtered.pass.ranked.vcf.gz",
```
Comment (ashwini06) [inline BALSAMIC/workflows/balsamic.smk:265]:
```suggestion
if config['analysis']["analysis_type"] in ["paired", "single"] and config["analysis"]["sequencing_type"] != "wgs":
```
```suggestion
    analysis_specific_results.extend([expand(vep_dir + "{vcf}.all.filtered.pass.ranked.vcf.gz",
```
Isnt it this file `.pass.ranked.vcf.gz` needs to be generated for wgs cases?

File: BALSAMIC/workflows/balsamic.smk (hunk 1 above)
Comment (hassanfa) [reply BALSAMIC/workflows/balsamic.smk:190]: oops, that why tests are failing. It is a new file. I'll add right away.

File: BALSAMIC/workflows/balsamic.smk (hunk 2 above)
Comment (hassanfa) [reply BALSAMIC/workflows/balsamic.smk:265]: WGS is still mixed with SNV/SV, I thought to add this for panel first. Then add it to WGS runs as well.

**Comment by hassanfa** [comment]
Successfully downloaded and added rank score: 
image

**Review by ashwini06 — COMMENTED** [review]
👍

**Pull request merged**